contracts/src/internal/HyperdriveLP.sol

```diff
@@ -200,7 +205,7 @@ abstract contract HyperdriveLP is
             true
         );
 
-        // Ensure that the spot APR is close enough to the previous weighted
+        // Ensure that the spot APR is less than the circuit close enough to the previous weighted
```

Comment:
The words in this comment aren't making sense in my head

Reply:
I consider explicitly enumerating these cases in the comment.  This is one of the most complicated conditionals in the code so it warrants an explanation